Formula/w/wgpu-native.rb

```diff
@@ -0,0 +1,39 @@
+class WgpuNative < Formula
+  desc "Native WebGPU implementation based on wgpu-core"
+  homepage "https://github.com/gfx-rs/wgpu-native"
+  url "https://github.com/gfx-rs/wgpu-native.git",
+      tag:      "v25.0.2.1",
+      revision: "af9074edf144efe4f1432b2e42c477429c4964c1"
+  license "Apache-2.0"
+  head "https://github.com/gfx-rs/wgpu-native.git", branch: "trunk"
+
+  depends_on "rust" => :build
+  depends_on "cmake" => :test
+  depends_on "ninja" => :test
+
+  def install
+    # Not using `cargo install` because wgpu-native doesn't ship binaries
```

Comment:
```suggestion
  depends_on "ninja" => :test
  uses_from_macos "llvm" => :build

  def install
    ENV["LIBCLANG_PATH"] = Formula["llvm"].opt_lib.to_s if OS.linux?
    # Not using `cargo install` because wgpu-native doesn't ship binaries
```
I think this should help the Linux build failure. Though maybe the `=> :build` part for LLVM may need to be dropped; unsure. But CI will complain if it's needed at runtime.

Reply:
Let's try it...